client/components/domains/use-your-domain-step/index.jsx

```diff
@@ -0,0 +1,371 @@
+	};
+
+	static defaultProps = {
+		analyticsSection: 'domains',
+		onSave: noop,
+	};
+
+	state = this.getDefaultState();
+
+	getDefaultState() {
+		return {
+			authCodeValid: null,
+			domain: null,
+			domainsWithPlansOnly: false,
+			inboundTransferStatus: {},
+			precheck: get( this.props, 'forcePrecheck', false ),
+			searchQuery: this.props.initialQuery || '',
+			submittingAuthCodeCheck: false,
+			submittingAvailability: false,
+			submittingWhois: get( this.props, 'forcePrecheck', false ),
+			supportsPrivacy: false,
+		};
+	}
+
+	getMapDomainUrl() {
+		const { basePath, mapDomainUrl, selectedSite } = this.props;
+		if ( mapDomainUrl ) {
+			return mapDomainUrl;
+		}
+
+		let buildMapDomainUrl;
+		const basePathForMapping = endsWith( basePath, '/use-your-domain' )
+			? basePath.substring( 0, basePath.length - 16 )
+			: basePath;
+
+		buildMapDomainUrl = `${ basePathForMapping }/mapping`;
+		if ( selectedSite ) {
+			const query = stringify( { initialQuery: this.state.searchQuery.trim() } );
+			buildMapDomainUrl += `/${ selectedSite.slug }?${ query }`;
+		}
+
+		return buildMapDomainUrl;
+	}
+
+	goToMapDomainStep = event => {
+		event.preventDefault();
+
+		this.props.recordMappingButtonClickInUseYourDomain( this.props.analyticsSection );
+
+		page( this.getMapDomainUrl() );
+	};
+
+	getTransferDomainUrl() {
+		const { basePath, transferDomainUrl, selectedSite } = this.props;
+
+		if ( transferDomainUrl ) {
+			return transferDomainUrl;
+		}
+
+		let buildTransferDomainUrl;
+		const basePathForTransfer = endsWith( basePath, '/use-your-domain' )
+			? basePath.substring( 0, basePath.length - 16 )
+			: basePath;
+
+		buildTransferDomainUrl = `${ basePathForTransfer }/transfer`;
+
+		if ( selectedSite ) {
+			const query = stringify( { initialQuery: this.state.searchQuery.trim() } );
+			buildTransferDomainUrl += `/${ selectedSite.slug }?${ query }`;
+		}
+
+		return buildTransferDomainUrl;
+	}
+
+	goToTransferDomainStep = event => {
+		event.preventDefault();
+
+		this.props.recordTransferButtonClickInUseYourDomain( this.props.analyticsSection );
+
+		page( this.getTransferDomainUrl() );
+	};
+
+	goBack = () => {
+		this.props.goBack();
+	};
+
+	getTransferPriceText = () => {
+		const {
+			cart,
+			currencyCode,
+			translate,
+			domainsWithPlansOnly,
+			isSignupStep,
+			productsList,
+			selectedSite,
+		} = this.props;
+		const { searchQuery } = this.state;
+		const productSlug = getDomainProductSlug( searchQuery );
+		const domainsWithPlansOnlyButNoPlan =
+			domainsWithPlansOnly && ( ( selectedSite && ! isPlan( selectedSite.plan ) ) || isSignupStep );
+
+		let domainProductPrice = getDomainPrice( productSlug, productsList, currencyCode );
+
+		if ( isNextDomainFree( cart ) || isDomainBundledWithPlan( cart, searchQuery ) ) {
+			domainProductPrice = translate( 'Free with your plan' );
+		} else if ( domainsWithPlansOnlyButNoPlan ) {
+			domainProductPrice = translate( 'Included with paid plans' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Included in paid plans' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).